support-frontend/assets/components/subscriptionCheckouts/layoutStyles.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import { css } from '@emotion/core';
+import { from } from '@guardian/src-foundations/mq';
+import { space } from '@guardian/src-foundations';
+import { neutral } from '@guardian/src-foundations/palette';
+
+export const mainCss = css`
+  max-width: ${space[280]}px;
```

Comment:
Does this work? This isn't how I understand `space`, but I may have misunderstood!

Reply:
No it didn't! Have fixed it.